packages/js/product-editor/src/components/attribute-control/edit-attribute-modal.tsx

```diff
@@ -185,6 +250,10 @@ export const EditAttributeModal: React.FC< EditAttributeModalProps > = ( {
 							</div>
 						) }
 					</div>
+
+					{ Boolean( additions || deletions ) && (
+						<Notice>{ getNoticeMessage() }</Notice>
+					) }
```

Comment:
I believe that if we don't verify that we're under the `Variations` tab, there's a chance we might display the message while editing `Attributes` in the `Organization` tab

![Screen Capture on 2023-09-28 at 15-19-25](https://github.com/woocommerce/woocommerce/assets/1314156/8047c418-b159-4143-94bb-2b62c342d6a4)

Reply:
Ohh!! good catch. Let me fix this asap!